### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Jimilian**
- fix formatting for json
- add examples for json files
- mention support of Gerrit Trigger Plugin

### CODE REVIEW

**Comment by Jimilian** [comment]
@sschuberth, please, take a look.

File: README.md [hunk 1]
```diff
@@ -55,23 +55,47 @@ To get started:
 1. Add "Post Build Action" -> "Send to Fluentd".
 1. Specify the desired file (and extension in JSON format if needed).
 
-Examples
+Send to Fluentd Post-build Action
```
Comment (sschuberth) [inline README.md]: I'm not sure what this is supposed to mean. Maybe "Sending data to Fluentd as a post-build action"?

File: README.md [hunk 2]
```diff
@@ -55,23 +55,47 @@ To get started:
...
 ---
 
-Plugin supports two types of JSON data: Single JSON and JSON array. Also, the plugin supports environment variable
+Using this plugin you can send custom data to your FluentD instance. Plugin supports two types of JSON data: Single JSON and JSON array. Also, the plugin supports environment variable
```
Comment (sschuberth) [inline README.md]: The plugin supports [...]

File: README.md [hunk 3]
```diff
@@ -55,23 +55,47 @@ To get started:
...
+3. If you extend `{"a1": "b1"}` by `{"number": "$BUILD_NUMBER"}`, the plugin would expand the environment variable for each execution separately.
 So, for the first run the plugin would submit
-{"a1": "b1", "number": "1"}
+`{"a1": "b1", "number": "1"}`
 and for second one
-{"a1": "b1", "number": "2"}
+`{"a1": "b1", "number": "2"}`
 etc.
 
+Plugin supports only json files with flat structure. It means that you can not have nested elements. See examples below.
```
Comment (sschuberth) [inline README.md]: The plugin supports [...]

File: README.md [hunk 4]
```diff
@@ -55,23 +55,47 @@ To get started:
...
+
+or
+
+`[{"test_name": "some_test", "duration": 100, "passed": false }, {"test_name": "another_test", "duration": 10, "passed": true }]`
+
+**Bad json**:
+
+`{"test_suit": "some_suite", "test": [{ "name": "test1"}, {"name": "test2"}]}`
+
 Additionally, the plugin adds a "timestamp" field into the JSON data. The value of this field is equals the start time of the job.
```
Comment (sschuberth) [inline README.md]: Even if not part of this change: Either "insert ... into" or "add ... to".

File: README.md [hunk 5]
```diff
@@ -55,23 +55,47 @@ To get started:
...
 
+Supported plugins
+---
+
+If you had installed one of this plugins, you can enable sending data provided by plugin:
```
Comment (sschuberth) [inline README.md]: "Sending the data provided by this plugin requires one of the following plugins to be installed:"

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Jimilian) [reply README.md]: @sschuberth, I fixed all notes except this one. It's optional dependencies and this plugins are not required.

**Review by sschuberth — APPROVED** [review]
